### ISSUE TITLE
[Feature Request]: Add a `latest-release` tag to the docker image

### ISSUE THREAD

**Issue by DanielWeigl** [grokability/snipe-it#12853]
### Is your feature request related to a problem? Please describe.

Currently each commit on master triggers a new tag on your docker image and releases dont have a dedicated tag. We have an automated update check on our cluster (keel) which automatically depolys the most recent version if a tag gets updated.

### Describe the solution you'd like

Can you add [here](https://github.com/snipe/snipe-it/blob/develop/.github/workflows/docker.yml#L34) and [here](https://github.com/snipe/snipe-it/blob/develop/.github/workflows/docker-alpine.yml#L34) something along:

```
type=semver,pattern=v{{major}}-latest
```
(untested)

So that its possible to subscribe to the most recent major version tag.

### Additional context

(based on documentaion from https://github.com/docker/metadata-action#typesemver )

**Comment by snipe** [comment]
> and releases dont have a dedicated tag

They do though, so I'm not sure what problem we're solving here. 

https://hub.docker.com/r/snipe/snipe-it/tags

You can def see the master and dev builds, but under that, you see the tagged releases. 

<blockquote><div><strong><a href="https://hub.docker.com/r/snipe/snipe-it/tags">Docker</a></strong></div></blockquote>

**Comment by DanielWeigl** [comment]
Yes, but its not possible to use one tag (eg. `v6-latest` or `latest-release`) to get the most recent release. If you follow the tag `master` or `latest` we get an update for each push to master, which not necessarily is a release. And if you subscribe to an explicit release tag (`v6.1.0`) we would never get automated updates.

**Comment by jerm** [comment]
This was fixed in https://github.com/snipe/snipe-it/pull/12992

Closing.
